app/utils/cookie.js

```diff
@@ -0,0 +1,32 @@
+export const getCookie = name => {
```

Comment:
@remkst777 
Rome, did not we have a utility that works with cookies?

Reply:
@sergeyi23 I couldn't find in the project files

Reply:
we had, but it wasn't used and it was deleted